### PR TITLE
Added section to include instructions on installing repo tool on Ubuntu 20.04 and higher version.

### PR DESCRIPTION
**Pull request by sanathusk**
Added section to include instructions on installing repo tool on Ubuntu 20.04 and higher version as repo package is not present in their repositories.

### CODE REVIEW

File: porting/first-steps.rst [hunk 1]
```diff
@@ -70,6 +70,36 @@ Install the required dependencies::
      python-markdown libxml2-utils xsltproc zlib1g-dev:i386 schedtool \
      repo liblz4-tool bc lzop imagemagick libncurses5 rsync
 
+.. Note::
+    repo is not available in repositories of Ubuntu 20.04 or later versions. Hence it has to be installed manually.
+
+
+Installing repo 
+^^^^^^^^^^^^^^^^^
```
Comment (doniks) [inline porting/first-steps.rst:78]: trim to the length of the headline please. Also I guess this should be a level deeper, ie inside Ubuntu section, not on same level as Ubuntu and Arch sections

File: porting/first-steps.rst [hunk 2]
```diff
@@ -70,6 +70,36 @@ Install the required dependencies::
...
+
+To install repo, we need to ensure that you curl and git is installed. To install curl & git run the following command :
+
+.. code-block::
```
Comment (doniks) [inline porting/first-steps.rst:82]: Did you see that travis CI check has failed? i think this line should be

.. code-block:: shell

File: porting/first-steps.rst [hunk 3]
```diff
@@ -70,6 +70,36 @@ Install the required dependencies::
...
+
+    sudo apt install curl && sudo apt install git
```
Comment (doniks) [inline porting/first-steps.rst:84]: sudo apt install curl git

would be easier. Actually, you aren't using git though. Actually, actually, we just installed curl (and git) four lines above, so I think we could drop this whole paragraph

File: porting/first-steps.rst [hunk 4]
```diff
@@ -70,6 +70,36 @@ Install the required dependencies::
      python-markdown libxml2-utils xsltproc zlib1g-dev:i386 schedtool \
      repo liblz4-tool bc lzop imagemagick libncurses5 rsync
 
+.. Note::
+    repo is not available in repositories of Ubuntu 20.04 or later versions. Hence it has to be installed manually.
```
Comment (doniks) [inline porting/first-steps.rst:74]: Let's make it more explicit:

The ``repo`` command is not ... 

that's double backticks around the word repo

File: porting/first-steps.rst [hunk 5]
```diff
@@ -70,6 +70,36 @@ Install the required dependencies::
...
+
+
+Installing repo 
+^^^^^^^^^^^^^^^^^
+
+To install repo, we need to ensure that you curl and git is installed. To install curl & git run the following command :
```
Comment (doniks) [inline porting/first-steps.rst:80]: ``repo``

File: porting/first-steps.rst [hunk 6]
```diff
@@ -70,6 +70,36 @@ Install the required dependencies::
...
+
+.. code-block::
+
+    sudo apt install curl && sudo apt install git
+
+Run the following commands to create a .bin directory in the home directory, and include it in your path.
```
Comment (doniks) [inline porting/first-steps.rst:86]: Mhm ... this is the first time I see the usage of `.bin` with a dot. I'd have a preference for `bin`. In a  quick internet search I find a couple mentions of ``bin``, e.g., https://help.ubuntu.com/community/HomeFolder , but I haven't easily found one with ``.bin``. But, I guess it's not really a standard. Do you feel strongly about the hidden dot.bin folder? If not, pls change it to bin.

**Review by doniks — CHANGES_REQUESTED** [review]

File: porting/first-steps.rst (hunk 1 above)
Comment (sanathusk) [reply porting/first-steps.rst]: Corrected as part of next commit.

File: porting/first-steps.rst (hunk 3 above)
Comment (sanathusk) [reply porting/first-steps.rst]: Corrected as part of next commit.

File: porting/first-steps.rst (hunk 4 above)
Comment (sanathusk) [reply porting/first-steps.rst]: Corrected as part of next commit

File: porting/first-steps.rst (hunk 5 above)
Comment (sanathusk) [reply porting/first-steps.rst]: Corrected as part of next commit

File: porting/first-steps.rst (hunk 6 above)
Comment (sanathusk) [reply porting/first-steps.rst]: Corrected as part of next commit

**Comment by sanathusk** [comment]
@doniks I have made the suggested changes.Can you review & merge the same.

File: porting/first-steps.rst (hunk 2 above)
Comment (sanathusk) [reply porting/first-steps.rst]: Corrected as part of next commit.

File: project/Related-projects.rst [hunk 7]
```diff
@@ -35,6 +35,12 @@ Ubuntu Touch
 `Ubuntu Touch <https://ubports.com>`_ is based on Unity 8 and is developed by UBports. It is the continuation of the earlier Ubuntu Touch project from Canonical. UBports is actively working on making their future releases available on Halium.
 
 
+hybris-mobian
+-------------
+
+`hybris-mobian <https://hybris-mobian.org/>`_ is a GNU/Linux distribution based on top of Mobian, a Debian-based distribution for mobile devices.The goal of hybris-mobian is to be able to run Mobian on Android phones
```
Comment (doniks) [inline project/Related-projects.rst:41]: space before The
dot after phones

**Review by doniks — CHANGES_REQUESTED** [review]

File: project/Related-projects.rst (hunk 7 above)
Comment (sanathusk) [reply project/Related-projects.rst]: @doniks The suggested changes have been committed & pushed.Please review  & merge the PR.

**Review by doniks — APPROVED** [review]

**Comment by doniks** [comment]
thanks @sanathusk

**Pull request merged**